kong/globalpatches.lua

```diff
@@ -424,6 +424,12 @@ return function(options)
       end
 
       toip = client.toip
+
+      -- Patch DNS query instrumentation
```

Comment:
```suggestion
      -- DNS query is lazily patched, it will only be wrapped
      -- when instrumentation module is initialized later and
      -- `opentelemetry_tracing` includes "dns_query" or set
      -- to "all".
```

Reply:
Thanks!